### PR TITLE
Fix RemoveConnectionFromAllGroups 

### PR DESCRIPTION
**Pull request by xingsy97**
### Summary of the changes
 - Separate `RemoveConnectionFromAllGroups` from `RemoveConnectionFromGroup`
 - Before this PR, `...FromAllGroups` was implemented by calling `...FromGroup` with `group=null`

### CODE REVIEW

File: src/Microsoft.Azure.SignalR.Management/HubContext/GroupManager.cs [hunk 1]
```diff
@@ -12,5 +12,7 @@ public abstract class GroupManager : IGroupManager
         public abstract Task AddToGroupAsync(string connectionId, string groupName, CancellationToken cancellationToken = default);
 
         public abstract Task RemoveFromGroupAsync(string connectionId, string groupName, CancellationToken cancellationToken = default);
+
+        public abstract Task RemoveFromAllGroupsAsync(string connectionId, CancellationToken cancellationToken);
```
Comment (Y-Sindo) [inline src/Microsoft.Azure.SignalR.Management/HubContext/GroupManager.cs:16]:
```suggestion
        public abstract Task RemoveFromAllGroupsAsync(string connectionId, CancellationToken cancellationToken = default);
```

File: src/Microsoft.Azure.SignalR.Management/HubContext/GroupManagerAdapter.cs [hunk 2]
```diff
@@ -10,14 +10,18 @@ namespace Microsoft.Azure.SignalR.Management
     internal class GroupManagerAdapter : GroupManager
     {
         private readonly IGroupManager _groupManager;
+        private readonly IHubLifetimeManager _lifetimeManager;
 
-        public GroupManagerAdapter(IGroupManager groupManager)
+        public GroupManagerAdapter(IGroupManager groupManager, IHubLifetimeManager lifetimeManager)
```
Comment (Y-Sindo) [inline src/Microsoft.Azure.SignalR.Management/HubContext/GroupManagerAdapter.cs:15]: Remove `IGroupManager` and implements all the methods with `IHubLifetimeManager`?

File: src/Microsoft.Azure.SignalR.Management/WebsocketsHubLifetimeManager.cs [hunk 3]
```diff
@@ -29,6 +29,11 @@ public override Task RemoveFromGroupAsync(string connectionId, string groupName,
                 throw new ArgumentException(NullOrEmptyStringErrorMessage, nameof(connectionId));
```
Comment (Y-Sindo) [inline src/Microsoft.Azure.SignalR.Management/WebsocketsHubLifetimeManager.cs:29]: This method can be deleted?

Comment (xingsy97) [reply src/Microsoft.Azure.SignalR.Management/WebsocketsHubLifetimeManager.cs]: updated

File: src/Microsoft.Azure.SignalR.Management/HubContext/GroupManagerAdapter.cs (hunk 2 above)
Comment (xingsy97) [reply src/Microsoft.Azure.SignalR.Management/HubContext/GroupManagerAdapter.cs]: updated

File: src/Microsoft.Azure.SignalR.Management/HubContext/GroupManager.cs (hunk 1 above)
Comment (xingsy97) [reply src/Microsoft.Azure.SignalR.Management/HubContext/GroupManager.cs]: updated

**Review by Y-Sindo — APPROVED** [review]

**Pull request merged**